src/Service/S3/tests/Integration/S3ClientTest.php

```diff
@@ -300,9 +302,9 @@ public function testPutObjectAcl(): void
                 'Grants' => [new Grant([
                     'Grantee' => new Grantee([
                         'DisplayName' => 'me',
-                        'Type' => 'canonical',
+                        'Type' => Type::CANONICALUSER,
```

Comment:
Bug spotted :)

Reply:
Are you sure?

Reply:
The const name is generated. So, yes, we ca choose whatever we want :)
Note: the value of the constant remain the same and is the value expected by AWS.

Maybe it's not obvious for `CANONICALUSER`, I found that `AMAZON_CUSTOMER_BY_EMAIL` is most readable than `AMAZONCUSTOMERBYEMAIL`

I fixed it.

Reply:
yes. The right value is `CanonicalUser`